modules/get-started/pages/release-notes/operator.adoc

```diff
@@ -26,21 +105,25 @@ spec:
     useFlux: true
 ----
 
+If you upgrade to v2.4.x from an older version in which Flux was already enabled, Flux will remain in use. If Flux was disabled previously, it will stay disabled after an upgrade unless you explicitly enable it.
```

Comment:
```suggestion
If you upgrade to v2.4.x from an older version in which Flux was explicitly enabled, Flux will remain in use. If Flux was explicitly disabled previously, it will stay disabled after an upgrade unless you explicitly enable it.
```